server/plugins/content.ts

```diff
@@ -0,0 +1,26 @@
+// This plugin is needed to solve the current bad implementation of NuxtContent
```

Comment:
This is beautiful. Thank you for your creativity @raar1

Reply:
Thanks @raar1 for all your hard work! I tested it via a local rendering and it is working perfectly. So, instead of:
```
---
title: Software Testing
type: slides
order: 1
---

---

<!-- .slide: data-state="title" -->

## Software Testing

---
```
The correct way to format the slide `md` file now is:
```
---
title: Software Testing
type: slides
order: 1
---

<!-- .slide: data-state="title" -->

## Software Testing

---
```
Please, correct me if I am missing something.

Reply:
Yes, that's right - I made the necessary changes in <https://github.com/esciencecenter-digital-skills/research-software-support/pull/82>. (I think this is correct? I don't recall needing the first separator before any slides in reveal.js but maybe I am wrong and my regex is faulty)

Reply:
OK, now there is a small difference. After Jaro's suggested change, we are now using `===` for the slides separator instead of `---`

Reply:
Thanks! I see. I will also fix it in NEBULA-docs...